### PR TITLE
[EDU-3210] add improvements to runtime and node compatibility PT

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue:

[EDU-3210](https://aziontech.atlassian.net/browse/EDU-3210)

Changes:

Add a table for polyfills.
Revamp built-in info.

[EDU-3210]: https://aziontech.atlassian.net/browse/EDU-3210?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/compatibilidade/node.mdx [hunk 1]
```diff
@@ -9,48 +9,41 @@ menu_namespace: runtimeMenu
...
-        requestId.exit(async () => {
-            logAsyncContext("log do callback de saída");
-        });
-        return new Response("ok");
-    });
-}
-addEventListener("fetch", (event) => {
-    event.respondWith(handleRequest(event.request));
-});
+A compatibilidade entre o **Azion Edge Runtime** e **Node.js** é uma tarefa em andamento, mas um conjunto de APIs do **Node Runtime** são compatíveis com o Azion Edge Runtime. A seguinte lista de APIs pode ser importada dentro das [edge functions](/pt-br/documentacao/produtos/guias/edge-functions/primeiros-passos/) com a sintaxe de importação padrão:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/compatibilidade/node.mdx:15]:
```suggestion
A compatibilidade entre o **Azion Edge Runtime** e **Node.js** é um trabalho em andamento, mas um conjunto de APIs do **Node Runtime** já é compatível com o Azion Edge Runtime. A seguinte lista de APIs pode ser importada dentro das [edge functions](/pt-br/documentacao/produtos/guias/edge-functions/primeiros-passos/) com a sintaxe de importação padrão:
```

Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/devtools/azion-edge-runtime/compatibilidade/node.mdx:15]: está falando sobre o conjunto...

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/compatibilidade/node.mdx [hunk 2]
```diff
@@ -9,48 +9,41 @@ menu_namespace: runtimeMenu
...
+
+```js
+import { API } from "import-origin";
 ```
 
+| API | Nível de Suporte | Exemplo de Código | Comentários |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/compatibilidade/node.mdx:21]:
```suggestion
| API | Nível de suporte | Exemplo de código | Comentários |
```

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/compatibilidade/node.mdx [hunk 3]
```diff
@@ -9,48 +9,41 @@ menu_namespace: runtimeMenu
...
+|---------|---------------|----------|-------------|
+| async_hooks | 🟡 Parcialmente suportada  | [Acessar o exemplo de código](https://github.com/aziontech/azion-samples/tree/dev/samples/node-async-hooks) | Apenas [AsyncLocalStorage](https://nodejs.org/api/async_context.html#class-asynclocalstorage) e [AsyncResource](https://nodejs.org/api/async_hooks.html#class-asyncresource) estão implementadas. |
+
+---
+
+## Node polyfills
+
+Um polyfill é um trecho de código, frequentemente usado em JavaScript, que traz novos recursos para ambientes que não possuem essas capacidades. Os polyfills são usados durante o tempo de *build* e podem ser configurados através do arquivo [vulcan.config.js](/pt-br/documentacao/devtools/vulcan/config/).
+Aqui está a lista de APIs do Node suportadas através de polyfills:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/compatibilidade/node.mdx:30]:
```suggestion
Um polyfill é um trecho de código, frequentemente usado em JavaScript, que traz novos recursos para ambientes que não possuem essas capacidades. Os polyfills são usados durante o tempo de *build* e podem ser configurados através do arquivo [vulcan.config.js](/pt-br/documentacao/devtools/vulcan/config/).

Aqui está a lista de APIs do Node suportadas através de polyfills:
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/compatibilidade/node.mdx [hunk 4]
```diff
@@ -9,48 +9,41 @@ menu_namespace: runtimeMenu
...
+
+| API | Nível de Suporte | Exemplo de Código |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/compatibilidade/node.mdx:32]:
```suggestion
| API | Nível de suporte | Exemplo de código |
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**